src/components/FormInput/formInput.component.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import './formInput.styles.scss';
+
+const FormInput = ({labelText, placeholder, onEmailInputChange, ...otherInputProps}) => {
```

Comment:
Why did you added ...otherInputProps?

Reply:
in case someone else might use the form input elsewere in the app and they need to add more classes to the input.

Reply:
I guess that makes sense. You can leave it.